src/diffusers/loaders.py

```diff
@@ -1400,6 +1414,11 @@ def unload_lora_weights(self):
             else:
                 self.unet.set_default_attn_processor()
 
+            if self.unet.aux_state_dict_populated:
+                for _, module in self.unet.named_modules():
+                    if hasattr(module, "old_forward") and module.old_forward is not None:
+                        module.forward = module.old_forward
+
```

Comment:
Should we also reset the `aux_state_dict_populated` after LoRA unloads since that state is now not true.

```suggestion
            if self.unet.aux_state_dict_populated:
                for _, module in self.unet.named_modules():
                    if hasattr(module, "old_forward") and module.old_forward is not None:
                        module.forward = module.old_forward
                self.unet.aux_state_dict_populated = False

```